templates/misc/message.part.html

```diff
@@ -121,25 +121,22 @@
                         </li>
                     {% endif %}
                 {% endif %}
-                {% if not is_mp %}
-                    {% if message.is_visible != False or perms_change %}
-                        <li>
-                            <a href="#signal-message-{{ message.id }}" class="ico-after alert open-modal">
+                {% if not is_mp and message.is_visible != False %}
```

Comment:
Ta condition n'est pas bonne, parce que ça cache le bouton pour tout le monde, y compris le staff. Il faut juste "not is_mp".